File: Leetcode-Solutions/Strings/mergeStringsAlternately.py
class Solution(object):
    def mergeAlternately(self, word1, word2):
        """
        :type word1: str
        :type word2: str
        :rtype: str
        """
        len1,len2=len(word1),len(word2)
        merged=''
        maxSize=max(len1,len2)

        for i in range(maxSize):
            if i<len1:
                merged+=word1[i]
            if i<len2:
                merged+=word2[i]
        return merged

File: Leetcode-Solutions/Strings/mergeStringsAlternately.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    const first=word1.length
    const sec=word2.length
    let merged=''
    let maxLen=Math.max(first,sec)
    for(let i=0; i<maxLen; i++){
       if(i<first){
        merged+=word1[i]
       } 
         if(i<sec){
        merged+=word2[i]
       } 
    }
    return merged
};
